### PR TITLE
feat: Seperate service view

### PR DESCRIPTION
**Pull request by johnjcsmith**
Extract "service" details onto a seperate page, making a place to introduce service metrics in a future PR.

image

image

### CODE REVIEW

File: admin/app/clusters/[clusterId]/LiveTables.tsx [hunk 1]
```diff
@@ -96,6 +98,45 @@ export function LiveTables({
 
   return (
     <div>
+      <div className="mt-12">
```
Comment (johnjcsmith) [inline admin/app/clusters/[clusterId]/LiveTables.tsx:101]: Add service cards

File: admin/app/clusters/[clusterId]/LiveTables.tsx [hunk 2]
```diff
@@ -175,41 +216,6 @@ export function LiveTables({
           noDataMessage="No services with function calls have been detected in the cluster lately."
         />
       </div>
-
-      <div className="mt-12">
```
Comment (johnjcsmith) [inline admin/app/clusters/[clusterId]/LiveTables.tsx:179]: Remove function details

File: admin/app/clusters/[clusterId]/page.tsx [hunk 3]
```diff
@@ -2,10 +2,10 @@ import { client } from "@/client/client";
 import { auth } from "@clerk/nextjs";
```
Comment (johnjcsmith) [inline admin/app/clusters/[clusterId]/page.tsx:2]: Renamed the `[id]` param to `[clusterId]`

File: admin/app/clusters/[clusterId]/ClusterLiveTables.tsx [hunk 4]
```diff
@@ -149,67 +190,6 @@ export function LiveTables({
           ]}
         />
       </div>
-      <div className="mt-12">
```
Comment (johnjcsmith) [inline admin/app/clusters/[clusterId]/ClusterLiveTables.tsx:152]: "services" and "executions" moved to service page

**Review by nadeesha — APPROVED** [review]
Nice!

**Pull request merged**